alliander/Issues/connection/description.xml

```diff
@@ -0,0 +1,50 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Electricity/Gas connection</en>
+		<nl>Demo Aansluiting</nl>
+	</Name>
+	<ShortName>
+		<en>Demo Connection</en>
+		<nl>Demo Aansluiting</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>alliander</IssuerID>
+	<CredentialID>connection</CredentialID>
+	<Description>
+		<en>Identification numbers (EAN) for your electricity and gas connection. See https://www.eancodeboek.nl/</en>
+		<nl>Identificatienummers (EAN) voor uw electriciteits- en gasaansluiting. See https://www.eancodeboek.nl/</nl>
+	</Description>
+
+	<Attributes>
+		<Attribute id="ean-gas">
```

Comment:
This would be the first occurence of a dash `-` in any attribute name; all other attributes so far just concatenate words, eg. [`pbdf.pbdf.idin.dateofbirth`](https://github.com/privacybydesign/pbdf-schememanager/blob/master/pbdf/Issues/idin/description.xml#L42).